app/controllers/admin/requests_controller.rb

```diff
@@ -0,0 +1,24 @@
+class Admin::RequestsController < AdminController
+  before_action :set_current_request, except: :index
+
+  def index
+    @requests = Request.available.includes(:user).page(params[:page]).per(50)
```

Comment:
`Request.available.includes(:user).page(params[:page]).per(50)` move to `requests` method